helpers/tf_cfg.py

```diff
@@ -17,7 +17,7 @@
 from rich import pretty
 from rich.logging import RichHandler
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger()
```

Comment:
You should not do that because any actions with this object can affect the others

Reply:
Actually, I have fixed the misconfiguration.
`tf_cfg.py` is an entry point for the logging and we configure it here. We need to configure the root logger (i.e. `logging.getLogger()`) once to easily reuse configuration by inheritance of the root logger in any module i.e. 
 `logging.getLogger(__name__)`. Example - this PR, logging works as expected and I just declared module-like logger by calling `logging.getLogger(__name__)` and I have all correct configuration presented.